addons/l10n_ch/models/ir_actions_report.py

```diff
@@ -58,10 +80,10 @@ def _render_qweb_pdf_prepare_streams(self, report_ref, data, res_ids=None):
                 invoice_stream = res[invoice_id]['stream']
                 writer = OdooPdfFileWriter()
                 writer.appendPagesFromReader(OdooPdfFileReader(invoice_stream, strict=False))
-                writer.appendPagesFromReader(OdooPdfFileReader(additional_stream['stream'], strict=False))
+                writer.appendPagesFromReader(OdooPdfFileReader(additional_stream, strict=False))
```

Comment:
I have the feeling you are breaking something here since `isr_inv_ids` is supposed to work and is returning something with a `stream` key. At line 73, you should set `{'stream': new_pdf_stream}` instead I think... to check